PersonSearchDetails/StepDefinitions/Person/PersonCreateStepDefinitions.cs

```diff
@@ -110,29 +136,63 @@ public void ThenTheExpectedAuditEventsWouldBeCreated()
+        public void WhenICreateTwoPeopleUsingTheSameDetails(string firstName, string dob, string dateMovedIn, string ethnicity, string gender, string preferredLanguage)
         {
             // generate a random string for surname, false or true sets the string to upper or lower case
             lastName = DHCWExtensions.RandomString(6, false);
-            // create our first person
-            PersonMethods.CreateBasicPerson(xrmBrowser, driver, firstname, dob, dateMovedIn, ethnicity, gender, preferredLanguage, lastName);
-            // create the 2nd person using the same details as above
-            PersonMethods.CreateBasicPerson(xrmBrowser, driver, firstname, dob, dateMovedIn, ethnicity, gender, preferredLanguage, lastName);
-            
+
+            for (int i= 0; i<2; i++)
+            {
+                //Create SharedNavigation method for "open new person"
+                xrmBrowser.Navigation.OpenSubArea("Workplace", "People");
+                xrmBrowser.CommandBar.ClickCommand("NEW PERSON");
+                //Switch back to main window after opening new person
+                driver.SwitchTo().Window(driver.WindowHandles.Last());
+
+                // select the correct iFrame - this should be refactored out in the future
+                driver.SwitchTo().Frame("contentIFrame1");
+
+
+
+                Page_PersonCoreDemographics.EnterFirstName(driver, firstName);
+                Page_PersonCoreDemographics.EnterLastName(driver, lastName);
+                Page_PersonCoreDemographics.EnterEthnicity(driver, ethnicity);
+                Page_PersonCoreDemographics.EnterPreferredLanguage(driver, preferredLanguage);
+                Page_PersonCoreDemographics.EnterGender(driver, gender);
+                Page_PersonCoreDemographics.EnterDateOfBirth(driver, dob);
+                Page_PersonCoreDemographics.EnterDateMovedIn(driver, dateMovedIn);
+
+                // save the record
+                //Note that we have had to use a custom save function elsewhere
+                xrmBrowser.CommandBar.ClickCommand("SAVE");
+                //xrmBrowser.ThinkTime(3000);
+                Console.WriteLine(lastName);
+            }
         }
 
 
         [Then(@"the duplicate detection rules will trigger")]
         public void ThenTheDuplicateDetectionRulesWillTrigger()
         {
+            //On the person page
             driver.SwitchTo().Frame("id = \"InlineDialog_Iframe\"");
             xrmBrowser.ThinkTime(1000);
+
+            //Locate the error element and pull back the text
+            //This needs a new name and some description
             String errorTextd = driver.FindElement(By.XPath("//*[@id=\"ErrorTitle\"]")).Text;
             xrmBrowser.ThinkTime(1000);
+
+            //Confirm Duplicate record is the text pulled back
+            //Query: Is this too brittle?
```

Comment:
I couldnt actually properly test this at the time, now that i know the duplication rules are in place for NHS Number i can use that to complete the test, i'll look at this as soon as i finish my scenario 8 bits